pkgs/top-level/all-packages.nix

```
@@ -6086,11 +6086,15 @@ with pkgs;
 
   jre17_minimal = callPackage ../development/compilers/openjdk/jre.nix {
     jdk = jdk17;
+    jlink = "${buildPackages.jdk17}/lib/*/bin/jlink";
   };
   jre21_minimal = callPackage ../development/compilers/openjdk/jre.nix {
     jdk = jdk21;
+    jlink = "${buildPackages.jdk21}/lib/*/bin/jlink";
+  };
+  jre_minimal = callPackage ../development/compilers/openjdk/jre.nix {
+    jlink = "${buildPackages.jdk}/lib/*/bin/jlink";
```

```suggestion
    jlink = "${buildPackages.jdk}/lib/openjdk/bin/jlink";
```

unless there's a benefit to the wildcards that i'm missing?

---

I think I added that to handle different JDK vendors, but I will check again.

---

Changed to just put the build JDK in nativeBuildInputs.